Frontend/ui-customer/src/Flow.purs

```diff
@@ -942,11 +970,27 @@ homeScreenFlow = do
                                       homeScreenFlow
             "TRIP_FINISHED"       -> do -- TRIP FINISHED
                                       void $ pure $ JB.exitLocateOnMap ""
-                                      if (getValueToLocalStore HAS_TAKEN_FIRST_RIDE == "false") then do
-                                        void $ pure $ metaLogEvent "ny_user_first_ride_completed"
-                                        (GetProfileRes response) <- Remote.getProfileBT ""
-                                        setValueToLocalStore HAS_TAKEN_FIRST_RIDE ( show response.hasTakenRide)
-                                        else pure unit
+                                      let appName = case getAppName "" of
```

Comment:
both code here and above looks like copy of each other, please write clean code and correct this